scopes/docs/docs/docs.ui.runtime.tsx

```diff
@@ -15,8 +15,8 @@ export class DocsUI {
   /**
    * register a new title badge into the overview section of a component.
    */
-  registerTitleBadge(...titleBadge: TitleBadge[]) {
-    this.titleBadgeSlot.register(titleBadge);
+  registerTitleBadge(titleBadges: TitleBadge[]) {
```

Comment:
this change might be a breaking change for anyone that only pass one badge not as array

Reply:
reverting
Out of interest, is there any advantage in doing it like this? Just that you can enter a single item without making it an array?

Reply:
Unreverted, as without this change couldn't register multiple badges.
It's not clear why, as both with and without the change the register function is passed an array of badges... Unclear what the difference is, but it just doesnt work with the previous syntax.
Probably something to do with arrays and maps...

Reply:
@benjgil - you need to use it like this:
```ts
docs.registerTitleBadge(firstBadge, secondBadge, etcBadge);
```

and it has the advantage of being backward compatible and easier to read/write

Reply:
please revert

Reply:
or allow single badges by checking `Array.isArray(badges)`

Reply:
we dont need backwards compatibility (i've changed the two places that currently used it) and the problem with your approach is that for whatever reason (myself and Gilad couldnt work it out with the short time we spent on it) it doesnt work when trying to add more than a single badge at a time.

Reply:
I dont see the utility of being able to add a single badge not as an array with one member, certainly not enough to refactor this again